src/promptflow/promptflow/connections.py

```diff
@@ -19,6 +19,8 @@
 
 @dataclass
 class BingConnection:
+    """Bing connection."""
```

Comment:
We may need to add more comments to this class.

Maybe add guidance on how to get the API key?